terraform/modules/public-load-balancer/main.tf

```diff
@@ -0,0 +1,79 @@
+#
+# Certificates
+#
+
+# TODO: use a single, ACM-managed cert with both domains on. There is already
+# such a cert in integration/staging/prod (but it needs defining in Terraform).
+data "aws_acm_certificate" "public_lb_default" {
+  domain   = "*.${var.public_lb_domain_name}"
+  statuses = ["ISSUED"]
+}
+
+data "aws_acm_certificate" "public_lb_alternate" {
+  domain   = "*.${var.app_domain}"
+  statuses = ["ISSUED"]
+}
+
+resource "aws_lb_listener_certificate" "service" {
+  listener_arn    = aws_lb_listener.public.arn
+  certificate_arn = data.aws_acm_certificate.public_lb_alternate.arn
+}
+
+#
+# Internet-facing load balancer
+#
+
+resource "aws_lb" "public" {
+  name               = "fargate-public-${var.app_name}-${var.cluster_name}"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.public_alb.id]
+  subnets            = var.public_subnets
+}
+
+resource "aws_lb_target_group" "public" {
+  name        = "${var.app_name}-${var.cluster_name}-public"
+  port        = 80
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+  target_type = "ip"
+
+  health_check {
+    path = "/healthcheck"
+  }
+
+  depends_on = [aws_lb.public]
```

Comment:
I have a feeling this might still needed but I can't remember for sure. Is there a Terraform bug we can link to? I did a quick search and couldn't find anything that fits this scenario. (Don't worry if not, just would be good to avoid mysterious-looking workarounds as they tend to then proliferate without being questioned.)

Reply:
I copied it over from Publisher - I've removed it since I also don't know the reason for it.